easybuild/easyconfigs/d/dbus-glib/dbus-glib-0.112-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'dbus-glib'
+version = '0.112'
+
+homepage = 'https://dbus.freedesktop.org/doc/dbus-glib'
+description = """D-Bus is a message bus system, a simple way for applications to talk to one another."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://dbus.freedesktop.org/releases/dbus-glib']
+sources = [SOURCE_TAR_GZ]
+checksums = ['7d550dccdfcd286e33895501829ed971eeb65c614e73aadb4a08aeef719b143a']
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('pkg-config', '0.29.2'),
```

Comment:
@sassy-crick Please use `pkgconf` instead, should be a drop-in replacement for `pkg-config`...